test/language/comments/hashbang-eval.js

```diff
@@ -0,0 +1,10 @@
+/*---
```

Comment:
I wish I didn't have to say that, but we need to have the 2 lines of copyright on each test file.

I'm sorry this is a thing we'd need to get fixed in a meeting. Thanks for understanding.

Reply:
I hate this requirement, btw.

Reply:
is it fine to have them below the `#!`?